recipes/wdiff/build.sh

```diff
@@ -0,0 +1,5 @@
+set -x
+
+./configure --prefix=${PREFIX} && \
+make && \
+make install
```

Comment:
Why do these need to be `&&` together? Can these just be separate lines?

Also is there a `make check`

Reply:
Yes, no need, just was recommended in README

I'll add `make check` and `make installcheck`